### PR TITLE
GEN-468 - chore: redirect [no|no-en|dk|dk-en]/new-member --> [no|no-en|dk|dk-en]/info

### PR DESCRIPTION
**Pull request by guilhermespopolin**
**This should wait for Peters 'go ahead'**

## Describe your changes

Redirect `new-member` routes to info pages. That following should apply:

`[no|no-en|dk|dk-en]/new-member --> [no|no-en|dk|dk-en]/info`

## Justify why they are needed

Part of _Home Coming_ project.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2537** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2537" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2537?utm_source=stack-comment).

**Review by robinandeer — COMMENTED** [review]
It seems like we also redirect `/se/new-member` - is that intended? Will it cause any issues?

**Comment by guilhermespopolin** [comment]
> It seems like we also redirect `/se/new-member` - is that intended? Will it cause any issues?

You're right. I thought it wouldn't be a problem but it is. Have changed.

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/next.config.js [hunk 1]
```diff
@@ -92,6 +92,16 @@ module.exports = withBundleAnalyzer({
       ],
     }
   },
+  async redirects() {
+    return [
+      {
+        source: '/(no|no-en|dk|dn-en)/new-member/:slug*',
+        destination: '/info',
```
Comment (alebedev) [inline apps/store/next.config.js:99]: Will it keep original locale?  Let's test

Comment (guilhermespopolin) [reply apps/store/next.config.js:99]: While testing I could verify that the original locale is kept. About the destination I need to double check tho.

Comment (robinandeer) [reply apps/store/next.config.js:99]: Yeah I think the flow is Redirect -> Not found on racoon -> Rewrite (to old hedvig-com)

**Review by robinandeer — COMMENTED** [review]
Hmmmm 🤔 now it redirects `/no/new-member` to `/info` (without locale)

Are we sure it's working correctly?

File: apps/store/next.config.js [hunk 2]
```diff
@@ -92,6 +92,34 @@ module.exports = withBundleAnalyzer({
       ],
     }
   },
+  async redirects() {
```
Comment (guilhermespopolin) [inline apps/store/next.config.js:95]: So I've change it as described in a comment:

1. Redirect accordingly
2. Leave for the rewrite definition, the last item of `fallback` to be more precise, to change the origin to market web.

However it's not working and I'm not sure how to debug redirects. Do you guys have an idea?
I've checked that we have `FALLBACK_ORIGIN_URL` configured for preview apps so I believe `fallback` redirects are being returned for it...

image

Comment (guilhermespopolin) [reply apps/store/next.config.js:95]: This is solved!

File: apps/store/src/utils/Features.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ const config = {
   MANYPETS_MIGRATION: process.env.NEXT_PUBLIC_FEATURE_MANYPETS_MIGRATION === 'true',
   SAS_PARTNERSHIP: process.env.NEXT_PUBLIC_FEATURE_SAS_PARTNERSHIP === 'true',
   INSURELY_CAR: process.env.NEXT_PUBLIC_FEATURE_INSURELY_CAR === 'true',
+  HOMECOMING: process.env.NEXT_PUBLIC_FEATURE_HOMECOMING === 'true',
```
Comment (robinandeer) [inline apps/store/src/utils/Features.ts:17]: guess this is technically not needed but why not ;)

**Review by robinandeer — APPROVED** [review]
Nice!

File: apps/store/src/utils/Features.ts (hunk 3 above)
Comment (guilhermespopolin) [reply apps/store/src/utils/Features.ts:17]: Yes, that's right. I can't use it in the config file but thought it could exist here as well.

**Pull request merged**